lib/bibliothecary/parsers/rubygems.rb

```diff
@@ -29,7 +29,7 @@ def self.mapping
       end
 
       def self.parse_gemfile_lock(manifest)
-        manifest.split("\n").map do |line|
+        manifest.each_line.map(&:chomp!).map do |line|
```

Comment:
We can probably ditch the second map, and probably with doing in the first line of the block so as not to pre-create the array.
```
manifest.each_line.map do |line|
  line = line.chomp!
  ...
end
```